frontend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:18-bullseye
+
+# Just install the packages; the source code will be loaded in later.
+RUN mkdir /opt/frontend
+ADD ./package.json /opt/frontend/
+WORKDIR /opt/frontend
+RUN npm install --production=false
```

Comment:
We've been using `yarn` instead `npm`, for what it's worth

Reply:
Updated!